pkg/usrlocalsharelima/usrlocalsharelima.go

```diff
@@ -27,19 +27,43 @@ import (
 // of the symlink, and not the actual executable. This is important when using Homebrew.
 //
 // If os.Args[0] is invalid, this function still falls back on os.Executable().
-var executableViaArgs0 = sync.OnceValues(func() (string, error) {
+var executableViaArgs0 = sync.OnceValues(func() (self string, _ error) {
```

Comment:
Our linter rules don't allow naked returns, so there is no point in naming the return value.